base/gmp.jl

```diff
@@ -113,57 +115,56 @@ module MPZ
 #   and `add!(x, a) = add!(x, x, a)`.
 using Base.GMP: BigInt, Limb
 
-const mpz_t = Ptr{BigInt}
-const bitcnt_t = Culong
+const Mpz_t = Ref{BigInt}
```

Comment:
Sure will change. I initially found it confusing since it was not capitalized, same with the other alias.